app/policies/external_activity_policy.rb

```diff
@@ -30,7 +30,7 @@ def set_private_before_matedit?
   end
 
   def copy?
-    user.present?
+    not_anonymous?
```

Comment:
For completeness, we sometimes use the `policy(model)` helper in controllers. In that case `current_user` is also used. It is implicitly used by the helper provided by Pundit.